flag/service/guest/guest.go

```diff
@@ -1,7 +1,9 @@
 package guest
 
 import "github.com/giantswarm/kvm-operator/flag/service/guest/update"
+import "github.com/giantswarm/aws-operator/flag/service/guest/ssh"
```

Comment:
Can we please have a multi line import block? You may want to check your golang integration with your editor. There are tools who do this automatically for you. I am using Atom and `go-plus`. This in turn uses `goimports` to sort imports on safe.